CHANGELOG.md

```diff
@@ -1,5 +1,68 @@
 # Changelog
 
+## Pelion Edge 2.2 - January 2021
+
+The primary features in this release:
+* [edge-core] Updated Edge Core to [0.15.0](https://github.com/PelionIoT/mbed-edge/blob/master/CHANGELOG.md#release-0150-2021-1-12)
+  * The new FOTA update framework library will be supported on platforms - `imx8mmevk` and `uz3eg-iocc`, (Feb 2021 release) not on `raspberrypi3`. To compile with this library, add following Bitbake parameters to local.conf - `MBED_EDGE_CORE_CONFIG_FIRMWARE_UPDATE="ON"`, `MBED_EDGE_CORE_CONFIG_FOTA_ENABLE="ON"` and `MBED_EDGE_CORE_CONFIG_CURL_DYNAMIC_LINK="ON"`.
+  * The old firmware update library - Update Client (UC) hub is only supported on `raspberrypi3` platform. To enable that add following parameter to local.conf - `MBED_EDGE_CORE_CONFIG_FIRMWARE_UPDATE="ON"`.
+* [maestro] Updated maestro to [v2.9.0](https://github.com/PelionIoT/maestro/releases/tag/v2.9.0)
+  * Gateway capabilities - Allows gateways to advertise the features supported by the platform. Maestro utilizes Edge Core's GRM JSON-RPC APIs to add to the gateway device's LwM2M resources. The registered resources are added under Pelion's reserved FeatureMgmt LwM2M object - 33457 with 3 resources - 0 - featureID, 1 - enabled, 2 - config.
+  * Remote config management via LwM2M - FeatureMgmt config resource allows users to remotely view the current configuration of the feature and also push a config update using the LwM2M cloud service APIs. Maestro, on receiving an update, writes the content to the file path specified in the respective parameter config_filepath.
+  * Removed parsing and generation of self-signed certificates. Also removed the platforms rp200 and wwrelay which are no longer supported.
+  * Maestro traffic now routed through edge-proxy instead of directly connecting to the Pelion Cloud
+* [logrotate] Added bbappend file to install a configuration which rotates the logs of files under `/var/log`.
+* [wait-for-pelion-identity] Converted the node program to bash. Removed the generation of self-signed certificate.
+* [edge-proxy] Configured services gateway logs (maestro), gateway stats (maestro), relay-term, fluentbit http endpoint to route traffic through edge-proxy and not consume self-signed certificate.
+* [meta-nodejs] Removed the dependency on node v8.x. Upgraded the node packages to work with default nodejs version of Dunfell.
+* [relay-term] Upgraded relay-term to work with node v12.x. Established an independent recipe from wwrelay-utils and removed the dependency on global-node-modules.
+* [fluentbit] Added recipe to install Fluentbit 1.3.5 on the gateway for providing an open source Log Processor and Forwarder solution.
```

Comment:
Should this suggest that it's an option?

Reply:
Ping @deepakpoornachandra

Reply:
We decided to disable maestro gateway log by default and only enable Fluentbit.